File: 프로그래머스/lv1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.kt
class Solution {
    fun solution(numbers: IntArray): IntArray {
        val list = mutableListOf<Int>()
        var number = 0

        for(i in 0 until numbers.size-1) {
            for(j in i+1 until numbers.size) {
                number = numbers[i] + numbers[j]
                if(number !in list) {
                    list.add(number)
                }
            }
        }

        return list.sorted().toIntArray()
    }
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.swift
func solution(_ numbers:[Int]) -> [Int] {
    var answer: [Int] = []
    
    for i in 0...numbers.count-2 {
        for j in i+1...numbers.count-1 {
            if !answer.contains(numbers[i] + numbers[j]) {
                answer.append(numbers[i] + numbers[j])
            }
        }
    }
    
    return answer.sorted()
}